twikit/client/client.py

```
@@ -2064,13 +2064,21 @@ async def get_latest_timeline(
-                continue
+        def handle_item(item):
             tweet = tweet_from_data(self, item)
-            if tweet is None:
-                continue
-            results.append(tweet)
+            if tweet is not None:
+                results.append(tweet)
+
+        for item in items:
+            if 'items' in item['content']: # home-conversation entries
+                for item in item['content']['items']:
```

Ruff (0.8.2)

Loop control variable `item` overrides iterable it iterates

(B020)